pallets/stake/src/lib.rs

```diff
@@ -308,15 +548,14 @@ decl_storage! {
 					"Stash does not have enough balance to bond."
 				);
 				let _ = if let Some(nominated_val) = opt_val {
-					<Module<T>>::nominate(
+					<Module<T>>::join_nominators(
 						T::Origin::from(Some(actor.clone()).into()),
 						nominated_val.clone(),
 						balance,
 					)
 				} else {
-					<Module<T>>::join_candidates(
+					<Module<T>>::join_candidates_no_fee(
```

Comment:
This changes the default fee for validators registered at genesis to 0%. It was previously 2%.

We could also add a `DefaultValidatorFee` config constant and use that here with the old `join_candidates` method. Then, there's no need to create the `join_candidates_no_fee` runtime method, but it does add a config constant and there are a lot already.

Reply:
Never mind, changed this so didn't add a new runtime method. The default fee was still changed to 0% for validators registered at genesis. is this a suitable default?